From: Simon McVittie <smcv@debian.org>
Date: Thu, 2 Apr 2020 12:52:16 +0100
Subject: build: Generate gdk.gresource.xml in sorted order

The order in which the resources get embedded matters for reproducible
builds. In the Meson build system, gen-gdk-gresources-xml already sorts
the list, but in the Autotools build system they were previously taken
in readdir() order.

Signed-off-by: Simon McVittie <smcv@debian.org>
Applied-upstream: 3.24.18, commit:b35cd705a752979e1a7ca0745b1a015326826c32
---
 gdk/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gdk/Makefile.am b/gdk/Makefile.am
index 479d6a1..3635864 100644
--- a/gdk/Makefile.am
+++ b/gdk/Makefile.am
@@ -471,7 +471,7 @@ stamp-gc-h: $(top_builddir)/config.status
 # Resources
 #
 
-glsl_sources := $(wildcard $(srcdir)/resources/glsl/*.glsl)
+glsl_sources := $(sort $(wildcard $(srcdir)/resources/glsl/*.glsl))
 
 gdk.gresource.xml: Makefile.am
 	$(AM_V_GEN) echo "<?xml version='1.0' encoding='UTF-8'?>" > $@; \
